opus/application/static_media/js/opus.js

```diff
@@ -713,6 +712,8 @@ var opus = {
         opus.prefs.widgets = [];
         o_widgets.updateWidgetCookies();
 
+        // set these to the current hash on opus init
+        [opus.lastSelections, opus.lastExtras] = o_hash.getSelectionsExtrasFromHash();
```

Comment:
Could you add the assignment we discussed making force_load = true so it's obvious what's going on?

Reply:
added w/comment